### DISCUSSION TITLE
workflow/mkdocs/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/workflow#19] [General]
# workflow/mkdocs/

https://shanenull.com/workflow/mkdocs/

**Comment by shane0** [comment]
another option that meets some checkboxes
https://app.gitbook.com/

**Reply by shane0** [reply]
https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#embedding-external-files

**Reply by shane0** [reply]
``` title=".browserslistrc"
--8<-- "include.md"
```

**Comment by shane0** [comment]
https://docs.asciinema.org/getting-started/#__tabbed_1_4

**Comment by shane0** [comment]
- grids https://squidfunk.github.io/mkdocs-material/reference/grids/

**Comment by shane0** [comment]
# solarized

| Name    | HEX       | RGB                  |
| ------- | --------- | -------------------- |
| Base03  | `#002b36` | `rgb(0, 43, 54)`     |
| Base02  | `#073642` | `rgb(7, 54, 66)`     |
| Base01  | `#586e75` | `rgb(88, 110, 117)`  |
| Base00  | `#657b83` | `rgb(101, 123, 131)` |
| Base0   | `#839496` | `rgb(131, 148, 150)` |
| Base1   | `#93a1a1` | `rgb(147, 161, 161)` |
| Base2   | `#eee8d5` | `rgb(238, 232, 213)` |
| Base3   | `#fdf6e3` | `rgb(253, 246, 227)` |
| Yellow  | `#b58900` | `rgb(181, 137, 0)`   |
| Orange  | `#cb4b16` | `rgb(203, 75, 22)`   |
| Red     | `#dc322f` | `rgb(220, 50, 47)`   |
| Magenta | `#d33682` | `rgb(211, 54, 130)`  |
| Violet  | `#6c71c4` | `rgb(108, 113, 196)` |
| Blue    | `#268bd2` | `rgb(38, 139, 210)`  |
| Cyan    | `#2aa198` | `rgb(42, 161, 152)`  |
| Green   | `#859900` | `rgb(133, 153, 0)`   |
